calculator/calculator.py

```diff
@@ -0,0 +1,218 @@
+"""Calculator module.
+
+This module allows evaluation of strings which represent mathematical
+expressions. The supported operations are declared in operators dictionary.
+
+Example:
+    To use this module, import the evaluate() function:
+
+        from calculator import evaluate
+
+    The parameter of this function is a mathematical expression (str). For
+    example, "1 + (3^-2) * (4! - 5$ 2)".
```

Comment:
When running this code a calculator should start that you can enter how many exercises you want, I don't want it to only be an imported function but a working program